config/vufind/KohaRest.ini

```diff
@@ -89,9 +89,9 @@ acceptTermsText = "I accept the terms in any language."
 ; VuFind item statuses
 [ItemStatusMappings]
 ;Item::Held = "Held"
-; Custom "Not For Loan" statuses can be mapped by appending the authorized value to
-; the status key like this (first one is negative authorized value, second one is
-; positive):
+; Custom "Not For Loan" or "Lost" statuses can be mapped by appending the authorized
+; value to the status key like this (first one is negative authorized value, second
+; one is positive):
 ;Item::NotForLoan-1 = "Ordered"
 ;Item::NotForLoan4 = "At Bindery"
```

Comment:
Would it make sense to add a Lost example here, just for clarity?

Reply:
Sure, I'll add.